src/pybind/mgr/cephadm/services/monitoring.py

```diff
@@ -417,7 +418,8 @@ def generate_config(
             'node_exporter_sd_url': node_exporter_sd_url,
             'alertmanager_sd_url': alertmanager_sd_url,
             'haproxy_sd_url': haproxy_sd_url,
-            'ceph_exporter_sd_url': ceph_exporter_sd_url
+            'ceph_exporter_sd_url': ceph_exporter_sd_url,
+            'nfs_exporter_sd_url': nfs_exporter_sd_url
```

Comment:
out of curiosity, what makes this nfs-_exporter_, as opposed to just `nfs_sd_url` like we do for haproxy or alertmanager. Afaik the other two that use this naming scheme are ceph-exporter and node-exporter which actually are being used for daemons with exporter in the name.

Reply:
Yeah I can skip exporter part. I think I added given we're adding nfs exporter job to prometheus target

Reply:
Shall I replace `nfs-exporter` with simply `nfs`? Given we don't have service name by `nfs-exporter` & just `nfs`